src/api/__tests__/api.unit.test.ts

```diff
@@ -1,81 +1,351 @@
 import "core-js";
 import mockAxios from "jest-mock-axios";
-import { API_ROOT, getApiUrl, apiGet } from "../api";
-import ERRORS from "../../shared/errors";
-import { makeTestError } from "../../shared/test-utils";
+import {
+  retrieveFactory,
+  getFactory,
+  createFactory,
+  patchFactory,
+  deleteFactory,
+  endpointFactory,
+  unwrap,
+} from "../api";
+import {
+  MOCK_RESPONSE,
+  MOCK_ERROR,
+  MOCK_BODY,
+  MOCK_CSRF,
+  MOCK_PARAMS,
+} from "../../shared/test-utils";
 
-describe("getApiUrl", () => {
-  it("returns root endpoint on empty string", () => {
-    expect(getApiUrl("")).toBe(API_ROOT);
+describe("listFactory", () => {
```

Comment:
Should be changed to getFactory to be consistent with naming (or, change getFactory to listFactory)

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/33/commits/464ee648dae35662078537b10ceff3002bf5bbf7